app/servers/query_master_parsing.py

```
@@ -0,0 +1,740 @@
+
+    if not parts:
+        logger.debug("No parsable parts found.")
+        return
+
+    # Debug: Show all parts
+    logger.debug("All Parts After Splitting:")
+    for i, part in enumerate(parts):
+        try:
+            decoded = part.decode("utf-8", errors="ignore")
+            logger.debug("Part %d: %s -> '%s'", i, part, decoded)
+        except:
```

Ruff (0.14.10)

Do not use bare `except`

(E722)